website/src/app/kb/authenticate/entra/readme.mdx

```diff
@@ -3,33 +3,31 @@ import PlanBadge from "@/components/PlanBadge";
 import Image from "next/image";
 import Link from "next/link";
 
-<PlanBadge plans={["enterprise"]}>
+<PlanBadge plans={["starter", "team", "enterprise"]}>
 
-# SSO + Sync with Microsoft Entra ID
+# SSO with Microsoft Entra ID
 
 </PlanBadge>
 
 Firezone integrates with
 [Microsoft Entra ID](https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-id)
 using a custom connector that supports both authentication and directory sync.
 Use this guide if you're looking to setup SSO with Microsoft Entra ID for your
-Firezone Enterprise account and want to automatically sync users and groups from
-Microsoft Entra ID to Firezone.
+Firezone Enterprise account and optionally sync users and groups from Microsoft
+Entra ID to Firezone.
```

Comment:
Perhaps it's just me, but when I read this I get the impression that it's only for setting up EntraID with a Firezone Enterprise account.  Specifically this line:
```
Use this guide if you're looking to setup SSO with Microsoft Entra ID for your Firezone Enterprise account...
```

What do you think about something along the lines of:
```
This guide will help you setup SSO with Microsoft Entra ID, as well as optionally setup directory sync if you are on the Firezone Enterprise plan.
```

Reply:
Ah I'll update this, missed that.

Reply:
I just removed the word `Enterprise` because I note immediately below in a callout:

> Directory sync is support for the **Enterprise** plan only.